.github/workflows/release-pr.yaml

```diff
@@ -119,34 +121,31 @@ jobs:
       - name: cleanup integration tests (cleanup)
         run: ./scripts/test-env.sh cleanup
   oldversion-integration-test:
+    timeout-minutes: 30
```

Comment:
We rather use the CI level env variable for setting it consistently in other repos